backends/events/sdl/sdl-events.cpp

```diff
@@ -1071,6 +1153,15 @@ bool SdlEventSource::handleControllerButton(const SDL_Event &ev, Common::Event &
 			{ EVENT_KEYDOWN, KeyState(KEYCODE_KP6, 0), EVENT_KEYDOWN, KeyState(KEYCODE_KP3, 0) }
 	};
 
+	if (!shouldGenerateMouseEvents()) {
+		event.type = buttonUp ? Common::EVENT_JOYBUTTON_UP : Common::EVENT_JOYBUTTON_DOWN;
```

Comment:
What about (for the moment) still allow holding R to slow slow down mouse here```if (ev.cbutton.button == SDL_CONTROLLER_BUTTON_RIGHTSHOULDER) {
	// Right shoulder is the modifier button that makes the mouse go slower
	_km.modifier = !buttonUp;
}```

Reply:
Pegasus Prime already uses the R button to open the Biochip panel, so this will need to be mapped to another button.

Reply:
Maybe just leave the code as is then. I guess if an engine with joystick support wants slow mouse hotkey it can implement it by itself...

Reply:
As per discord discussion, we could use holding down X to slow down mouse here.

So:
```
if (!shouldGenerateMouseEvents()) {
	if (ev.cbutton.button == SDL_CONTROLLER_BUTTON_X) {
		// Right shoulder is the modifier button that makes the mouse go slower
		_km.modifier = !buttonUp;
	}
(...)
```

Reply:
Why would you want to keep a specific key pressed down in order to slow down the mouse movement? It's bad practice to reserve a specific key for this kind of functionality, as that key can be used in specific games, and slowing down mouse movement may not be needed in all games.

Reply:
I agree